connector-common/src/test/java/com/redhat/cloud/notifications/connector/ConnectorRoutesTest.java

```diff
@@ -114,6 +105,7 @@ protected void testSuccessfulNotification() throws Exception {
 
         JsonObject incomingPayload = buildIncomingPayload(targetUrl);
         remoteServerMockEndpoint.expectedMessagesMatches(checkOutgoingPayload(incomingPayload));
+        remoteServerMockEndpoint.expectedMessageCount(1);
```

Comment:
We have to add an `expectedMessageCount` to run `expectedMessageMatches` checks as expected.

Reply:
I'm curious. What happens if `expectedMessageCount` is missing?

Reply:
Thanks for the details!